.tekton/pull-request.yaml

```diff
@@ -59,6 +59,14 @@ spec:
         workspaces:
           - name: workspace
             workspace: workspace
+      - name: sast-unicode-check
+        runAfter:
+          - fetch-repository
```

Comment:
Will this work actually?  I think the `upload` step takes `IMAGE_URL` and `IMAGE_DIGEST`, which are not known until the `build-container` task finishes.

Reply:
Updated, change it to 
```
        runAfter:
          - build-image-index
 ` ``